### ISSUE TITLE
[BUG] kontainer-engine-driver-lke consistently fails to start

### ISSUE THREAD

**Issue by lgarber-akamai** [rancher/rancher#44168]
**Rancher Server Setup**
- Rancher version: v2.8.1-head
- Installation option: Docker

**Information about the Cluster**
- Kubernetes version: N/A
- Cluster Type: LKE

**User Information**
- What is the role of the user logged in?: Admin

**Describe the bug**

`kontainer-engine-driver-lke` v0.0.8 fails to start with the following error:

```
/management-state/bin/kontainer-engine-driver-lke: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by /management-state/bin/kontainer-engine-driver-lke)
```

This error is raised because the release binaries for v0.0.8 were not built using `CGO_ENABLED=0`.

**To Reproduce**

Activate the Linode LKE cluster driver consuming kontainer-engine-driver-lke v0.0.8.

**Result**

The cluster driver hangs on "Activating" and the error above is repeatedly logged.

**Expected Result**

The LKE cluster driver runs as expected.

**Additional context**

This issue has been address in kontainer-engine-driver-lke v0.0.9 so a simple version bump should resolve the issue. I'll be putting up PRs to do this shortly 🙂

**Comment by snasovich** [comment]
This is to be validated by @lgarber-akamai FYI @daviswill2

**Comment by lgarber-akamai** [comment]
Confirmed that this issue is resolved in [rancher/rancher v2.9-head (793118f)](https://hub.docker.com/layers/rancher/rancher/v2.9-793118fc730d915f06d0cd2593509b67273c34e6-head/images/sha256-2a42cfe2d8e03cc944525a1b929e406e9ecf637e59e17e2c0e48932b406b2249?context=explore).

**Comment by snasovich** [comment]
Closing per the above comment. FYI @daviswill2
